pydarn/io/borealis/borealis_array.py

```diff
@@ -149,101 +184,43 @@ def arrays(self):
         """
         return self._arrays
 
+    @property
+    def borealis_version(self):
+        """
+        The version of the file, taken from the 'borealis_git_hash' field, in
+        the init.
+        """
+        return self._borealis_version
+
+    @property
+    def format_class(self):
+        """
+        The format class used for the file, from the borealis_formats module.
+        """
+        return self._format_class
+
     def read_file(self) -> dict:
         """
         Reads the specified Borealis file using the other functions for
         the proper file type.
 
         Reads the entire file.
 
-        See Also
-        --------
-        read_bfiq
-        read_rawacf
-        read_antennas_iq
-
         Returns
         -------
         arrays: dict
-            borealis data dictionary. Keys are data field names and
+            borealis data dictionary of arrays. Keys are data field names and
```

Comment:
``` suggestion
            Borealis data dictionary of arrays. Keys are data field names and
```

Might have missed some of these but do you prefer borealis or Borealis?

Reply:
changed all to Borealis with capital.